DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ if(NOT DEFINED FPGA_DEVICE)
                     \nPlease refer to the README for information on target selection.")
 else()
     # Check if the target is a BSP
-    if(IS_BSP MATCHES "1" OR FPGA_DEVICE MATCHES ".*de10_agilex.*")
+    if(IS_BSP MATCHES "1" OR FPGA_DEVICE MATCHES ".*(de10|ofs|ref).*")
```

Comment:
Please revert this part of the change to keep all the code samples cmake files identical
We can discuss doing a bulk edit for such a change